SimGeneral/DataMixingModule/plugins/DataMixingModule.cc

```diff
@@ -235,6 +236,13 @@ namespace edm
       produces< int >("bunchSpacing");
       produces<CrossingFramePlaybackInfoNew>();
 
+      std::vector<edm::InputTag> GenPUProtonsInputTags;
+      if( ps.exists("GenPUProtonsInputTags") )
```

Comment:
The data mixing is needed when running premixing, for which I have updated as part of this PR the configuration at:
SimGeneral/DataMixingModule/python/mixOne_simraw_on_sim_cfi.py
I could update similarly the configuration at:
​SimGeneral/​DataMixingModule/​python/​mixOne_sim_on_sim_cfi.py
Could you point me to which workflow uses it (so I can run a basic test)?
The other two files with configurations for "data on data" and "data on sim" do not apply in this case.
Thanks.